pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -206,6 +206,27 @@ in {
     };
   };
 
+  octoprint-ldap = buildPlugin rec {
+    pname = "OctoPrint-LDAP";
+    version = "1.1.0";
```

Comment:
Would make our lives easier: https://github.com/gillg/OctoPrint-LDAP/issues/16

Reply:
Thank you very much for creating the issue.  
Unfortunately I do not have any influence on its progress.